src/bacnet/basic/service/h_apdu.c

```diff
@@ -511,6 +513,20 @@ static bool apdu_confirmed_dcc_disabled(uint8_t service_choice)
                 status = true;
                 break;
         }
+    } else if (dcc_communication_initiation_disabled()) {
+        switch (service_choice) {
+            case SERVICE_CONFIRMED_DEVICE_COMMUNICATION_CONTROL:
+            case SERVICE_CONFIRMED_REINITIALIZE_DEVICE:
+            /* WhoIs will be processed and I-Am initiated as response. */
```

Comment:
No, Who-Is and I-Am are not confirmed services, despite their being paired together.

Please remove WHO_IS and I_AM from this switch statement as they will never be processed as 'confirmed' services.

Reply:
Removed Who-Is and I-Am are not confirmed services. Moved UnconfirmedAuditNotification to unconfirmed disabled check function.